### ISSUE TITLE
Corax54 ZMK config unavailable

### ISSUE THREAD

**Issue by jonathanforking** [dnlbauer/splitkeyboard#5]
Hi,
I wanted to configure the firmware ahead of my build, but the link to the ZMK repo in [corax54/README.md](https://github.com/dnlbauer/corax-keyboard/blob/main/corax54/README.md#firmware) points to a 404 page (perhaps it is set to private?).

This is the address I'm talking about:
https://github.com/dnlbauer/corax54-zmk-config

**Comment by jonathanforking** [comment]
I just realized that the config is available under https://github.com/dnlbauer/corax56-zmk-config/tree/corax54, will keep the issue open for the broken link

**Comment by dnlbauer** [comment]
Hi,

thanks for pointing this out. There is also https://github.com/dnlbauer/corax-zmk-config which I created to build the FW for both versions in the main repo. There is still an issue with non-working scrollwheels on this version, thought. My plan is to change the link to this repo once it's done. 

I understand it's quite confusing now. So thanks for creating this issue. I will let it open until I found a good way to provide both firmware versions.
